site/docs/reference/working-logs-stats.md

```diff
@@ -9,10 +9,20 @@ Access to statistics is still a work in progress. For now, this documentation de
 
 ## Accessing logs
 
-You can access logs by materializing them to an external endpoint, or from the command line.
+You can access logs in the web application, by materializing them to an external endpoint, or from the command line.
+
+### Accessing logs in the web application
+
+The Flow web application displays logs for each running catalog task.
```

Comment:
The logs that are currently shown in the UI are actually only the logs from the _publication_ (build). We don't currently have a way to display runtime logs in the UI.